src/Components/CustomMenu/CustomMenu.Component.jsx

```diff
@@ -25,11 +25,20 @@ export class CustomMenu extends React.Component {
 
   render() {
     return (
-      <div>
+      <div className="custom-menu">
         <SelectField
+          underlineStyle={{display: 'none'}}
+          iconStyle={{ fill: '#000000', marginRight: '1rem' }}
+          labelStyle={{ textAlign: 'right' }}
           value={this.state.value}
           onChange={this.handleChange}
-          style={{ display: 'block', fontSize: '0.8rem', textAlign: 'center', width: '10rem', padding: '0 auto' }}
+          style={
+          {
```

Comment:
Put this style outside the component in some variable and then use it here.

Reply:
@kimobrian Will look into this and make changes ASAP

Reply:
@kimobrian I've just researched this. Apparently, for material ui components, one has to style them inline. You can't style them from a separate stylesheet. Unless you mean that I declare a styles variable inside the render but above the return function like this?
<img width="862" alt="screen shot 2018-05-23 at 7 17 01 pm" src="https://user-images.githubusercontent.com/25130454/40437417-f7f2e09c-5ebd-11e8-9f22-6a8a964e4b03.png">

Reply:
Yes, that's what I meant. Declare it as a variable and then use it.

Reply:
Done